both/api/images/images.js

```diff
@@ -0,0 +1,52 @@
+import { Meteor } from 'meteor/meteor';
+import { Mongo } from 'meteor/mongo';
+import { SimpleSchema } from 'meteor/aldeed:simple-schema';
+
+import { PlaceInfos } from '../place-infos/place-infos';
+
+export const Images = new Mongo.Collection('Images');
+
+Images.schema = new SimpleSchema({
+  placeId: {
+    type: String,
+  },
+  hashedIp: {
+    type: String,
+  },
+  appToken: {
+    type: String,
+  },
+  timestamp: {
+    type: Date,
```

Comment:
Would like to have an ISO date in the JSON output – would you be okay with that?

Reply:
Done